src/index.tsx

```diff
@@ -75,24 +77,67 @@ export default class AppIntroSlider<ItemT = any> extends React.Component<
     showPrevButton: false,
     showSkipButton: false,
     bottomButton: false,
+    renderVertically: false,
+    autoPlay: false
   };
   state = {
     width: 0,
     height: 0,
     activeIndex: 0,
   };
   flatList: FlatList<ItemT> | undefined;
+  activeInterval: any;
+
+  componentDidMount() {
+    if (this.props.autoPlay) this.startScroll();
+  }
+
+  // Clear interval when user closes
+  componentWillUnmount() {
+    clearInterval(this.activeInterval);
+  }
+
+  startScroll() {
+    this.activeInterval = setInterval(this.scrolling, 2000);
+  }
+
+  scrolling = () => {
```

Comment:
```suggestion
  scroll = () => {
```
If we just call the function scroll it has consistent naming with `startScroll`

Reply:
okay i'll change name to scroll